app/scripts/lib/snap-keyring/snap-keyring.ts

```diff
@@ -76,6 +79,7 @@ export const snapKeyringBuilder = (
             try {
               await handleUserInput(confirmationResult);
               await getKeyringController().persistAllKeyrings();
+              getPreferencesController().setSelectedAddress(_address);
```

Comment:
We should remove the `_` prefix.

Reply:
Solved in [70c49de](https://github.com/MetaMask/metamask-extension/pull/21267/commits/70c49dee35749c2aed55965cb13bda50eab48ee7)